modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -28,8 +30,11 @@
 /**
  * Facebook Instant Article node wrapper class.  Builds up an InstantArticle
  * object using field formatters.
+ *
+ * Class DrupalInstantArticleDisplay
+ * @package Drupal\fb_instant_articles_display
  */
-class InstantArticleNodeWrapper {
+class DrupalInstantArticleDisplay {
```

Comment:
@scottrigby yeah – this is not really important btw – my thinking was that in the fully qualified name `Drupal\fb_instant_articles\DrupalInstantArticleDisplay` the class name just repeats the words that are already in the namespace, so it doesn't really tell you what the class does. `Drupal\fb_instant_articles\DisplayInstantArticle` has the same problem. That's why I thought `NodeAdapter` or something similar could make sense.

But.. We can completely just get back to this later on. No need to hold off on this and I see the PR was just merged in 👍